images/win/scripts/Installers/Run-Antivirus.ps1

```diff
@@ -0,0 +1,39 @@
+################################################################################
+##  File:  Run-Antivirus.ps1
+##  Desc:  Run a full antivirus scan.
+##         Run right after cleanup before we sysprep
+################################################################################
+
+try{
+    Update-MpSignature
+}
+catch {
+    Write-Host "Some error was found"
+}
+
+$run_scan_antivirus = $env:run_scan_antivitrus
```

Comment:
Please correct the spellcheck

Reply:
Also, since we use this variable only in the single place, I prefer to use just `$env:run_scan_antivirus` in condition

Reply:
Fixed typo. Good point @maxim-lobanov , thank you! Used just $env:run_scan_antivirus in if statement and removed variable $run_scan_antivirus.